### PR TITLE
feat: ColorPicker support defaultFormat

### LINKED ISSUE #44477: ColorPicker 颜色选择器，增加 defaultFormat 属性
### What problem does this feature solve?

用户能够设置默认的 format 属性值

### What does the proposed API look like?

ColorPicker 颜色选择器，增加 defaultFormat 属性

### PR DESCRIPTION
**Pull request by CYBYOB**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/44477

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 6372e3a</samp>

This pull request adds a new feature to the `ColorPicker` component, allowing the user to specify the initial color format. It also updates the component's documentation in Chinese to reflect the new prop.

### <samp>🤖 Generated by Copilot at 6372e3a</samp>

*  Add `defaultFormat` prop to `ColorPicker` component to allow user to specify default color format ([link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-be92dd92152a780003d48ddc95ec6716367d327f9f890f1221b84614aaa8168fR48), [link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-be92dd92152a780003d48ddc95ec6716367d327f9f890f1221b84614aaa8168fR77), [link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-be92dd92152a780003d48ddc95ec6716367d327f9f890f1221b84614aaa8168fR119), [link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-9d1ff106a032e484c8f982279c4e1295d23f69904a26e020e99310c446e93249R51))
  * Update `ColorPickerProps` interface in `components/color-picker/ColorPicker.tsx` to include optional `defaultFormat` prop ([link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-be92dd92152a780003d48ddc95ec6716367d327f9f890f1221b84614aaa8168fR48))
  * Pass `defaultFormat` prop to `useColor` hook in `components/color-picker/ColorPicker.tsx` to set initial color format state ([link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-be92dd92152a780003d48ddc95ec6716367d327f9f890f1221b84614aaa8168fR77))
  * Set `defaultValue` prop of `Select` component in `components/color-picker/ColorPicker.tsx` to `defaultFormat` prop to match selected format with default format ([link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-be92dd92152a780003d48ddc95ec6716367d327f9f890f1221b84614aaa8168fR119))
  * Update documentation for `ColorPicker` component in `components/color-picker/index.zh-CN.md` to include `defaultFormat` prop in API table ([link](https://github.com/ant-design/ant-design/pull/44487/files?diff=unified&w=0#diff-9d1ff106a032e484c8f982279c4e1295d23f69904a26e020e99310c446e93249R51))

### CODE REVIEW

**Comment by RedJue** [comment]
补个单测吧

**Comment by afc163** [comment]
为啥需要 defaultFormat ？

**Comment by RedJue** [comment]
> 为啥需要 defaultFormat ？

应该是想非受控模式下，设置默认的 format，现在想要实现这个得受控才能实现。

**Comment by CYBYOB** [comment]
> > 为啥需要 defaultFormat ？
> 
> 应该是想非受控模式下，设置默认的 format，现在想要实现这个得受控才能实现。

是的

**Comment by CYBYOB** [comment]
> 补个单测吧

好的，我看下~

**Comment by RedJue** [comment]
ping

**Comment by RedJue** [comment]
rebase 一下上游

**Comment by CYBYOB** [comment]
> rebase 一下上游

好的，我统一看下~

**Comment by li-jia-nan** [comment]
下周要发 5.9.0 了，看看这个能不能赶得上，可以一起上车

**Comment by CYBYOB** [comment]
还在写单测，到时看下吧~&nbsp;


陈元宝
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年9月3日(星期天) 中午11:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ant-design/ant-design] feat: ColorPicker support defaultFormat (PR #44487)





 
下周要发 5.9.0 了，看看这个能不能赶得上，可以一起上车

**Comment by RedJue** [comment]
单测进度咋样了@CYBYOB

File: components/color-picker/index.zh-CN.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ group:
 | arrow | 配置弹出的箭头 | `boolean \| { pointAtCenter: boolean }` | true | |
 | children | 颜色选择器的触发器 | React.ReactNode | - | |
 | defaultValue | 颜色默认的值 | string \| `Color` | - | |
+| defaultFormat | 颜色格式默认的值 | `rgb` \| `hex` \| `hsb` | - | 5.9.0 |
```
Comment (Wxh16144) [inline components/color-picker/index.zh-CN.md:51]: 英文文档不要遗漏了~

**Review by Wxh16144 — CHANGES_REQUESTED** [review]

File: components/color-picker/index.zh-CN.md (hunk 1 above)
Comment (RedJue) [reply components/color-picker/index.zh-CN.md:51]: 确实，我补一下

**Review by RedJue — APPROVED** [review]

**Review by Wxh16144 — APPROVED** [review]

**Pull request merged**